Bindings/Java/Matlab/Hopper_Device/RunInteractiveHopperSolution.m

```diff
@@ -141,13 +161,14 @@ function RunHopperGUI(varargin)
     hopper.addComponent(device);
     
     % Configure the device to wrap over the patella.
-   %debug1 = hopper.hasComponent([deviceNames{d}])
-    %debug2 = hopper.hasComponent(['device_' deviceNames{d}])
-    if patellaWrap{d} && hopper.hasComponent(['device_' deviceNames{d}])
+
+    if patellaWrap{d} && hopper.hasComponent([deviceNames{d}])
         if strcmp(deviceNames{d},'device_passive')
-            cable = PathSpring.safeDownCast(hopper.updComponent(['device_' deviceNames{d} '/cableAtoB']));
+            % TODO: Change to downcast from PathSpring when PathSpring is
+            % working for passive device
+            cable = PathActuator.safeDownCast(hopper.updComponent([deviceNames{d} '/cableAtoB']));
         elseif strcmp(deviceNames{d},'device_active')
-            cable = PathActuator.safeDownCast(hopper.updComponent(['device_' deviceNames{d} '/cableAtoB']));
+            cable = PathActuator.safeDownCast(hopper.updComponent([deviceNames{d} '/cableAtoB']));
```

Comment:
I will change this to reflect the patella wrapping code in RunHopperWithDevice, which works for both scripts. It was causing your patella wrapping issue above.